.circleci/config.templ.yml

```diff
@@ -934,6 +934,15 @@ jobs:
           wait: postgres
           pattern: 'aiopg'
 
+  avro:
+    <<: *contrib_job_small
+    docker:
+      - image: *ddtrace_dev_image
+      - *testagent
+    steps:
+      - run_test:
+          pattern: 'avro'
+
```

Comment:
Could we add this to GitLab since we're actively migrating from CircleCI -> GitLab right now? [This is an example PR of how it would work](https://github.com/DataDog/dd-trace-py/pull/10502/files)

Reply:
done